packages/app/app/components/AlbumView/index.js

```diff
@@ -154,6 +155,7 @@ class AlbumView extends React.Component {
           <div className={styles.album_buttons}>
             {this.renderPlayAllButton(album)}
             {this.renderOptionsButtons(album)}
+            <ArtistLink artistLink={this.props.album.uri} circular={true} size={'small'} />
```

Comment:
This is not exactly what I meant here. We need to check if this data is coming from Bandcamp, and use relevant links for merch/albums. We'll want an album link here (which looks ok) and the merchandise link on the artist page (which might have to be passed down from the metadata plugin in case it's not there yet).